### ISSUE TITLE
Bug in restAPI

### ISSUE THREAD

**Issue by lfdversluis** [Tribler/tribler#2226]
Was running tribler got the following stack:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 783, in __bootstrap
    self.__bootstrap_inner()
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
--- <exception caught here> ---
  File "/usr/lib/python2.7/dist-packages/twisted/python/threadpool.py", line 196, in _worker
    result = context.call(ctx, function, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/twisted/python/context.py", line 118, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/usr/lib/python2.7/dist-packages/twisted/python/context.py", line 81, in callWithContext
    return func(*args,**kw)
  File "/home/laurens/Documents/thesis/tribler/Tribler/Core/Modules/restapi/events_endpoint.py", line 57, in on_search_results_torrents
    "result": convert_db_torrent_to_json(torrent)}) + '\n')
  File "/home/laurens/Documents/thesis/tribler/Tribler/Core/Modules/restapi/util.py", line 20, in convert_db_torrent_to_json
    return {"id": torrent[0], "infohash": torrent[1].encode('hex'), "name": torrent[2], "size": torrent[3],
exceptions.KeyError: 0
```

**Comment by devos50** [comment]
I found out that this error occurs when remote search results from Dispersy are coming in (which have a different dictionary structure). I will fix this as soon as possible.

**Comment by devos50** [comment]
@whirm or @lfdversluis can you close this once (since it's fixed by #2264 )?
